core/src/com/unciv/logic/map/Perlin.kt

```diff
@@ -54,6 +56,26 @@ object Perlin {
         return total/max
     }
 
+    fun ridgedNoise3d(x: Double, y: Double, z: Double,
```

Comment:
I added a ridgedNoise function to try it but It would take work to do another kind of generator using it. It's not used right now and might not be useful. Maybe it is better to remove this function